libopenage/engine.cpp

```diff
@@ -374,24 +374,25 @@ void Engine::loop() {
 			// camera movement speed, in pixels per millisecond
 			// one pixel per millisecond equals 14.3 tiles/second
 			float mov_x = 0.0, mov_y = 0.0, cam_movement_speed_keyboard = 0.5;
+			bool has_focus = SDL_GetWindowFlags(this->window) & SDL_WINDOW_MOUSE_FOCUS;
 
 			input::InputManager &input = this->get_input_manager();
 			using Edge = input::InputManager::Edge;
 
 			if (input.is_down(SDLK_LEFT) or
-			    input.is_mouse_at_edge(Edge::LEFT, coord.window_size.x)) {
+			    input.is_mouse_at_edge(Edge::LEFT, coord.window_size.x) and has_focus) {
```

Comment:
Do we want to stop scrolling for keyboard presses also? I think that would be good.

In any case, I'd make some parenthesis to make it explicit.